content/review/content-tree.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Content Tree"
+metaTitle: "sensenet - Content Tree"
+metaDescription: "sensenet Content Tree"
+---
+
+sensenet content repository is basically a tree structure of the various stored content (e.g. folders, documents, user, tasks, etc.). A specific content is identified by a unique number (id) and also by its path. The root of the tree is at `/Root` path, all other content is placed somewhere under this root content - for example the users are placed at `/Root/IMS`.
+
+# Structured storage of content
+
+Tree structure lets you orginize, group your content easily by creating separate subtrees. It's not only important because of the possibility the configure content accessibility for different types of users on different parts of the tree, but also because of the inheritence of several things like content types' underlying business logic, metadata field configurations and settings.
```

Comment:
```suggestion
Tree structure lets you organize, group your content easily by creating separate subtrees. It's not only important because of the possibility to configure content accessibility for different types of users on different parts of the tree, but also because of the inheritence of several things like content types' underlying business logic, metadata field configurations and settings.
```